src/tools/PrizeTierController/PrizePoolTitle.tsx

```diff
@@ -11,31 +10,27 @@ import classNames from 'classnames'
  * @returns
  */
 export const PrizePoolTitle = (props: {
```

Comment:
`PrizeTierHistoryTitle`

Reply:
Displaying `V1` or `V2` in the future would also be quite helpful

Reply:
Renamed in ceae289